altlas/Assets/scripts/LeverScript.cs

```diff
@@ -3,10 +3,22 @@
 using UnityEngine;
 
 public class LeverScript : MonoBehaviour, ClickableInterface {
+
   public GlobeMovingScript globeMovingScript;
-  
-  void ClickableInterface.onClick(){
-    globeMovingScript.moving = true;
-    globeMovingScript.expanded = !globeMovingScript.expanded;
+
+    void ClickableInterface.onClick(){
+        GameObject mapInMiddle = MoveStack.MAP_ON_MIDDLE_OF_DESK;
+        if (mapInMiddle != null) {
```

Comment:
this linking logic will be changed by deus so that on map moving will return the game object. could you please it as him so we dont mix up to much logic